CorsixTH/Lua/world.lua

```diff
@@ -1205,13 +1205,8 @@ function World:onEndMonth()
     self:checkIfGameWon()
   end
 
-  -- Change population share for the hospitals, TODO according to reputation.
-  -- Since there are no competitors yet the player's hospital can be considered
-  -- to be fairly good no matter what it looks like, so after gbv.AllocDelay
-  -- months, change the share to half of the new people.
-  if self.month >= self.map.level_config.gbv.AllocDelay then
-    self:getLocalPlayerHospital().population = 0.5
-  end
+  local local_hospital = self:getLocalPlayerHospital();
```

Comment:
My understanding from the comment was that AllocDelay was simply simulating a reputation impact - it was just assumed after that period that you were doing so much better than the AI hospitals that you would receive half of the patients instead of the fair share (25%). Since the reputation impact means you can end up with up to 75% of the patients I don't think that's any value any more.

I feel that the map maker should increase the world spawn rate instead if that's the effect they are going for.